lua/caching_handlers.lua

```diff
@@ -53,6 +53,9 @@ function caching_handlers._post_request_callback(response, request_info, cacheab
     local cache_uri = caching_handlers._get_cache_uri(request_info)
     local success, err = xpcall(
         function()
+            -- Refresh cluster topology before writing the result
```

Comment:
is it not possible to have openresty do this outside of the scope of a request for us?